examples/train_pylate_contrastive.py

```diff
@@ -0,0 +1,69 @@
+import torch
+from datasets import load_dataset
+from sentence_transformers import (
+    SentenceTransformerTrainer,
+    SentenceTransformerTrainingArguments,
+)
+
+from pylate import evaluation, losses, models, utils
+
+# Add at the start of your train_pylate_contrastive.py
+import torch
+torch._inductor.config.fallback_random = True
+torch._inductor.config.triton.unique_kernel_names = True
+# Or completely disable torch compile
+# model.forward = torch.compile(model.forward, mode="max-autotune", fullgraph=True)
```

Comment:
```suggestion
# ModernBERT is compiled by default, so there is no need to call compile explicitly and it actually breaks the model
# model = torch.compile(model)
```

Reply:
So my suggestion is to remove everything altogether
If you want people to not try compiling the model and having the same issue as you, you can add this comment on top of the commented compile code to let them know.